cas/aciinfo.go

```diff
@@ -0,0 +1,70 @@
+package cas
+
+import (
+	"database/sql"
+	"time"
+)
+
+// ACIInfo is used to store informations about an ACI
+// BlobKey is the key in the blob and imageManifest store of the related ACI file.
```

Comment:
Better to just inline these docstrings, for example - http://golang.org/pkg/net/http/#Response